docroot/themes/humsci/humsci_basic/src/scss/components/_chosen.scss

```diff
@@ -173,55 +162,60 @@
   //
   // }
 }
-
 // Multi Chosen
-// .chosen-container-multi {
-//   .chosen-choices {
-//
-//   }
-//
-//   .chosen-choices li {
-//     &.search-field {
-//       input[type="text"] {
-//
-//       }
-//     }
-//
-//     &.search-choice {
-//
-//       span {
-//
-//       }
-//
-//       .search-choice-close {
-//
-//         &:hover {
-//
-//         }
-//       }
-//     }
-//
-//     &.search-choice-disabled {
-//
-//     }
-//
-//     &.search-choice-focus {
-//
-//       .search-choice-close {
-//
-//       }
-//     }
-//   }
-//
-//   .chosen-results {
-//
-//   }
-//
-//   .chosen-drop .result-selected {
-//
-//   }
-// }
+.chosen-container-multi {
```

Comment:
If you nest a new style for `.chosen-drop` under here you can get the dropdown to align with the input like the single chosen input does:

```
.chosen-drop {
    margin-top: hb-calculate-rems(-2px);
    border-radius: 0 0 hb-calculate-rems($hb-border-radius) hb-calculate-rems($hb-border-radius);
    background-clip: padding-box;
}
```
These are styles that are added via chosen.js for the single dropdown but not the multi for some reason.

Reply:
There are also some alignment issues when you select items:

<img width="565" alt="Screen Shot 2020-04-08 at 9 39 02 AM" src="https://user-images.githubusercontent.com/2486846/78790638-cff3fa00-797c-11ea-9650-c3fa6865d08b.png">